starfish/core/spots/DecodeSpots/trace_builders.py

```diff
@@ -68,7 +73,7 @@ def build_traces_sequential(spot_results: SpotFindingResults, **kwargs) -> Inten
 
     i = 0
     for (r, c), attrs in spot_results.items():
```

Comment:
would not call this attrs in this context because attrs and spot_attrs gets confusing.